sauna/plugins/ext/simple_domain.py

```diff
@@ -0,0 +1,34 @@
+import socket
+
+from sauna.plugins import Plugin, PluginRegister
+
+my_plugin = PluginRegister('SimpleDomain')
+
+
+@my_plugin.plugin()
+class SimpleDomain(Plugin):
+
+    @my_plugin.check()
+    def request(self, check_config):
+        domain = check_config.get('domain')
+
+        try:
+            r = socket.gethostbyname(domain)
```

Comment:
That'is why i call this plugin simple. For you what is the minimal features that i need to implement ?